File: examples/basic.py
from qbt_bridge import QuantumBridge
from qbt_bridge.providers import SimulatorProvider

bridge = QuantumBridge([SimulatorProvider(seed=7)])
print(bridge.connect())
packet = bridge.control_packet(shots=2048)
print(packet["quantum_mix"])
print(packet["states"][0]["provenance"])

File: rust/qbt-bridge/examples/basic.rs
use qbt_bridge::providers::SimulatorProvider;
use qbt_bridge::QuantumBridge;

fn main() -> Result<(), Box<dyn std::error::Error>> {
    let mut bridge = QuantumBridge::new(vec![Box::new(SimulatorProvider::new(7))]);
    println!("{:#?}", bridge.connect());

    let packet = bridge.control_packet(2048);
    println!("{}", serde_json::to_string_pretty(&packet)?);
    Ok(())
}
